BoltsSwift.xcodeproj/project.pbxproj

```diff
@@ -785,27 +805,31 @@
 			isa = XCBuildConfiguration;
 			baseConfigurationReference = 812DB5271D3597BF00552C9F /* BoltsSwift-iOS.xcconfig */;
 			buildSettings = {
+				SWIFT_VERSION = 5.0;
 			};
 			name = Debug;
 		};
 		87FEF37E1A9085FA00C60678 /* Release */ = {
 			isa = XCBuildConfiguration;
 			baseConfigurationReference = 812DB5271D3597BF00552C9F /* BoltsSwift-iOS.xcconfig */;
 			buildSettings = {
+				SWIFT_VERSION = 5.0;
 			};
 			name = Release;
 		};
 		87FEF3801A9085FA00C60678 /* Debug */ = {
 			isa = XCBuildConfiguration;
 			baseConfigurationReference = 812DB52B1D3597BF00552C9F /* BoltsSwiftTests-iOS.xcconfig */;
 			buildSettings = {
+				SWIFT_VERSION = 5.0;
 			};
 			name = Debug;
 		};
 		87FEF3811A9085FA00C60678 /* Release */ = {
 			isa = XCBuildConfiguration;
 			baseConfigurationReference = 812DB52B1D3597BF00552C9F /* BoltsSwiftTests-iOS.xcconfig */;
 			buildSettings = {
+				SWIFT_VERSION = 5.0;
```

Comment:
Hey @bpisano-luna this project uses xcconfigs to define the SWIFT_VERSION.
Please revert all "SWIFT_VERSION"-changes in this file and change the SWIFT_VERSION in all "Configurations/*xcconfig" files.

Reply:
Thanks for your reply. I missed this part. I've changed all the version to Swift 5.0 in the configs and also incremented the version number to 1.5.0